common/djangoapps/django_sudo_helpers/templates/sudo/third_party_auth_links.html

```diff
@@ -0,0 +1,17 @@
+{% load i18n third_party_auth_links %}
+<script type="text/javascript">
+    function thirdPartySignin(event, url) {
+      event.preventDefault();
+      window.location.href = url;
+    }
+</script>
+<span class="deco-divider" style="background: transparent;">
+    <span class="copy">{% trans "or" %}</span>
```

Comment:
This `trans` call does not give sufficient context to our human translators - please see https://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html